docs/Makefile

```diff
@@ -21,13 +21,19 @@ endif
 docs: cli
 	cd .. && mkdocs build --strict
 
+LATEST_RELEASE_META_URL=https://api.github.com/repos/k0sproject/k0s/releases/latest
```

Comment:
Instead of "latest" shouldn't we use the version given as input for the workflow?

Reply:
yeah, makes sense

Reply:
yes